lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -24,51 +33,59 @@ def self.replace_version_number(previous_version_number, new_version_number, fil
         end
       end
 
+      def self.determine_next_version_using_labels(repo_name, github_token, rate_limit_sleep)
+        old_version = Actions.sh("git describe --tags --abbrev=0").strip
+        UI.important("Determining next version after #{old_version}")
+        org = "RevenueCat"
```

Comment:
I've been hardcoding the `org` in all the new code I've been writing... I guess it doesn't hurt to make it flexible, but I don't think we will be using this in other org's repos and we avoid having an extra parameter around.